certification.md

```diff
@@ -1,62 +1,54 @@
-9. Channel manager cancels the created test reservations group
+## Pre-Certification
+
+1. Email integrations@mewssystems.com to request a Client token to be set up in the Mews [Test Environment](mews-api.md#test-environment).
+2. Create the following endpoint URLs for the Mews [Test Environment](mews-api.md#test-environment)
+    * [Update Availability](channel-manager-api.md#update-availability) 
+    * [Update Prices](channel-manager-api.md#update-prices)
+    * [Update Restrictions](channel-manager-api.md#update-restrictions)
+    * [Confirm Booking](channel-manager-api.md#confirm-booking) 
+    * [Change Notification](channel-manager-api.md#change-notification) \(optional\)
+3. Email the endpoints to integrations@mewssystems.com.
+4. Set up the connection in the channel manager system.
+5. Use the [Get Properties](mews-api.md#get-properties) call to receive the `connectionToken` for the [Test Hotel](mews-api.md#test-environment).
+6. Pull hotel information, space mapping and rate mapping information using the [Get Configuration](mews-api.md#get-configuration) call.
+7. Map the [Test Hotel](mews-api.md#test-environment) in the UI of the channel manager backend using the data received in the [Get Configuration](mews-api.md#get-configuration) call.
+
+**Note:** Multi-property Mews customers (vacation rentals, apartments, villas, etc.) are often set up in Mews as spaces of one Mews property.  E.g. Property is the Mews Customer and Spaces are the rental properties.  These spaces (properties) can be sent to the channel manager as multiple connections, 1 connection token per space (property) or as a single connection for the entire property manager.
+
+## Certification Tests
+
+These tests will be done during a 60-minute call with the Mews Integrations team and the channel manager.  
+
+| Test                             | Requirements                         |
+| -------------------------        | ------------------------------------ |
+| **Receive multi-space, multi-rate availability push** | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped from the [Test Hotel](mews-api.md#test-environment) and the [Update Availability](channel-manager-api.md#update-availability) endpoint is functional. |
+| **Receive multi-space, multi-rate rate push** | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped and the [Update Prices](channel-manager-api.md#update-prices) endpoint is functional. | 
+| **Receive multi-space, multi-rate restriction push** | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped and the [Update Restrictions](channel-manager-api.md#update-restrictions) endpoint is functional. | 
+| **Incorrect Space code error** | Connection returns an error in [plain response](general-remarks.md#plain-response) that includes the incorrect space code. |
+| **Unmapped Space code error** | Connection returns an error in [plain response](general-remarks.md#plain-response) that includes the unmapped space code. |
```

Comment:
what is difference between incorrect and unmapped? it is the same problem, no?

Reply:
If you are doing this way, you can also require the error code in those special cases.